.github/workflows/reusable-windows-msi.yml

```
@@ -25,3 +25,4 @@ jobs:
         persist-credentials: false
     - name: Build CPython installer
       run: .\Tools\msi\build.bat --doc -"${ARCH}"
```

I... guess this isn't valid bash now? Do we have to do this?

```suggestion
      run: ./Tools/msi/build.bat --doc -"${ARCH}"
```